Symfony/CS/Fixer.php

```diff
@@ -134,7 +134,7 @@ public function fix(ConfigInterface $config, $dryRun = false, $diff = false)
             $this->stopwatch->openSection();
         }
 
-        $fileCacheManager = new FileCacheManager($config->usingCache(), $config->getDir(), $config->getFixers());
+        $fileCacheManager = new FileCacheManager($config->usingCache(), $config->getCacheDir(), $config->getFixers());
```

Comment:
Cherry-pick this change into #1076 (which btw is pointless without this changes) and then close this PR.